modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -13,15 +13,15 @@ The following section describes known issues in {oadp-first} plugins:
 
 The velero plugin panics on imagestream backup. A missing secret causes this panic error.
 
-When the backup and the Backup Storage Location (BSL) are managed outside the scope of the Data Protection Application (DPA), the OADP controller, meaning the DPA reconciliation does not create the relevant `oadp-$BSL-registry-secret`.
+When the backup and the Backup Storage Location (BSL) are managed outside the scope of the Data Protection Application (DPA), the OADP controller, meaning the DPA reconciliation does not create the relevant `oadp-<backup storage location name>-registry-secret`.
 
 When the backup is run, the openshift-velero-plugin panics on the imagestream backup, with a panic error:
 
 [source,terminal]
 ----
 024-02-27T10:46:50.028951744Z time="2024-02-27T10:46:50Z" level=error msg="Error backing up item"
 backup=openshift-adp/$backupJobName error="error executing custom action (groupResource=imagestreams.image.openshift.io,
-namespace=$backedupNamespace, name=postgres): rpc error: code = Aborted desc = plugin panicked:
+namespace=<backup storage location name>, name=postgres): rpc error: code = Aborted desc = plugin panicked:
```

Comment:
don;t kill me
<BSL Name>

Reply:
`<backup storage location name>` changed to `<BSL Name>`